### ISSUE TITLE
Under or Over are not working, even with x-underline-at-descent-line set as true

### ISSUE THREAD

**Issue by chris-transloc** [ema2159/centaur-tabs#86]
If I set the bar as under or over it doesn't show up, only left is working. I've simplified my config for it down to this to rule out some other setting:

```
(use-package centaur-tabs
  :demand
  :custom
  (centaur-tabs-set-bar 'under)
  (x-underline-at-descent-line t)
  :config
  (centaur-tabs-mode t))
```

I'm using doom emacs, if that makes a difference. I've also tried setting x-underline-at-descent-line before calling use-package with no luck.

**Comment by ema2159** [comment]
Try setting the variables in the :config block usi g setq and see if thar gives you the same result.

**Comment by chris-transloc** [comment]
That did not work, but I did learn that if I toggle centaur-tabs-mode off and back on it shows up. Based on that I tried moving the (centaur-tabs-mode t) statement out to after the use-package command, but that didn't work.

**Comment by ema2159** [comment]
Also, you should only do (centaur-tabs-mode), the t is unnecessary

**Comment by chris-transloc** [comment]
I finally figured it out by using the `after-call` method that doom provides to tell it to only load after  the 'doom-load-theme-hook'. Apparently something doom does while loading themes has to happen first or the under/overlines don't appear.

**Comment by ema2159** [comment]
Awesome! I get a little bit lost sometimes because I'm not a Doom user. I'll try to use Chemacs to change between profiles and test this kind of stuff.

**Comment by chris-transloc** [comment]
Thanks. This package is one of my new favorites. It's amazing work. Thank you!

**Comment by ema2159** [comment]
Thank you very much for the kind words! I'm waiting for Emacs 27 to be released in order to integrate the package with the native tabs!

**Comment by JordanFaust** [comment]
@chris-transloc Can you expand on specifically what you did to correct that behavior (with code if possible)? Seeing the same on a fresh doom install.

**Comment by sgleizes** [comment]
I'm using doom as well, and since I upgraded from b172646b49, the under/over line is no longer working.
However it works after toggling centaur-tabs mode off and on, as described here, which suggest a loading order issue.
I've tried using `:after-call` and a variety of other approaches with no success.

My config is the following (on top of doom's):
```
(use-package! centaur-tabs
  :config
  (setq centaur-tabs-style "bar"
        centaur-tabs-set-bar 'under
        centaur-tabs-height 32
        centaur-tabs-set-icons t
        x-underline-at-descent-line t)
  (centaur-tabs-headline-match))
```
Any help is appreciated.
Thanks for the great package!
